### PR TITLE
Update trip list to reverse chronological order

### PR DESCRIPTION
**Pull request by mlbiche**

### CODE REVIEW

File: user/views.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def privacy():
         user = User.objects.get(id=current_user.id)
         return render_template('user/privacy.html',
                            user=user,
-                           locations=user.locations)
+                           locations=sorted(user.locations, key=lambda l: l.date, reverse=True))
```
Comment (vinyll) [inline user/views.py]: Locations are sorted by default on save.
I guess `locations=user.locations.reverse()` should be enough.

Comment (mlbiche) [reply user/views.py]: Oh sweet, way more efficient ! Thanks 🎉

File: user/views.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ def privacy():
         user = User.objects.get(id=current_user.id)
         return render_template('user/privacy.html',
                            user=user,
-                           locations=user.locations)
+                           locations=reversed(user.locations))
```
Comment (vinyll) [inline user/views.py:46]: Where does thie `reversed` function come from?

Comment (vinyll) [reply user/views.py:46]: Does this actually work?

Comment (mlbiche) [reply user/views.py:46]: This actually works ! After some research, I've found that the difference between `reversed` and `list.reverse` is that `reversed` returns the reversed list but leaves the original list unchanged while `list.reverse` reverses the list but does not return the result. To user `list.reverse`, I have to add the line `user.locations.reverse()` before the `return`. What do you prefer ?

More information about `reversed` : https://docs.python.org/fr/3/library/functions.html?highlight=reversed#reversed

Comment (vinyll) [reply user/views.py:46]: That's indeed simple and great!

**Review by vinyll — APPROVED** [review]

**Pull request merged**